features/src/main/feature/camel-features.xml

```diff
@@ -1160,6 +1160,13 @@
         <bundle dependency='true'>wrap:mvn:com.bazaarvoice.jolt/jolt-core/${jolt-version}</bundle>
         <bundle>mvn:org.apache.camel.karaf/camel-jolt/${project.version}</bundle>
     </feature>
+    <feature name='camel-joor' version='${project.version}' start-level='50'>
```

Comment:
should be after camel-jooq to respect alphabetical order

Reply:
I gotta revise the alphabet:D Fixed, thanks